ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/marketplace/minimal-permission/provision/cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-marketplace-minimal-permission-provision
+  steps:
+  - ref: azure-provision-service-principal-minimal-permission
+  - chain: ipi-conf-azure
```

Comment:
we can replace the following steps with `cucushift-installer-rehearse-azure-ipi-marketplace-provision`? Or even, we can skip to create the provision chain, but place them in install workflow yaml.

Reply:
make sense, updated.